sdk/tests/conformance/rendering/bind-framebuffer-flush-bug.html

```diff
@@ -0,0 +1,149 @@
+
+    const vertexBuffer = gl.createBuffer();
+    gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer);
+    gl.bufferData(gl.ARRAY_BUFFER, 32, gl.DYNAMIC_DRAW);
+    gl.bufferSubData(gl.ARRAY_BUFFER, 0, new Float32Array([
+        1, 1,
+        1, -1,
+        -1, 1,
+        -1, -1,
+    ]));
+    gl.vertexAttribPointer(0, 2, gl.FLOAT, false, 8, 0);
+    gl.enableVertexAttribArray(0);
+    gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffer);
+
+    const blendTextureProgram = gl.createProgram();
+    const blendTextureVertex = gl.createShader(gl.VERTEX_SHADER);
+    gl.shaderSource(blendTextureVertex, `
+        precision mediump float;
+        attribute vec2 pos;
+        varying vec2 uv;
+        void main(){
+            uv = (pos + vec2(1.0)) / 2.0;
+            gl_Position=vec4(pos*2.-1.,0.,1.);
+        }
+    `);
+    gl.compileShader(blendTextureVertex);
+    gl.attachShader(blendTextureProgram, blendTextureVertex);
+    const blendTextureFragment = gl.createShader(gl.FRAGMENT_SHADER);
+    gl.shaderSource(blendTextureFragment, `
+        precision mediump float;
+        uniform sampler2D texture;
+        varying vec2 uv;
+        void main(){
+            gl_FragColor=texture2D(texture, uv);
+        }
+    `);
+    gl.compileShader(blendTextureFragment);
+    gl.attachShader(blendTextureProgram, blendTextureFragment);
+    gl.linkProgram(blendTextureProgram);
+
+    const solidColorProgram = gl.createProgram();
+    const solidColorVertex = gl.createShader(gl.VERTEX_SHADER);
+    gl.shaderSource(solidColorVertex, `
+        precision mediump float;
+        attribute vec2 pos;
+        void main(){
+            gl_Position=vec4(pos,0.,1.);
+        }
+    `);
+    gl.compileShader(solidColorVertex);
+    gl.attachShader(solidColorProgram, solidColorVertex);
+    const solidColorFragment = gl.createShader(gl.FRAGMENT_SHADER);
+    gl.shaderSource(solidColorFragment, `
+        precision mediump float;
+        void main(){
+            gl_FragColor=vec4(0.,0.,1.,1.);
+        }
+    `);
+    gl.compileShader(solidColorFragment);
+    gl.attachShader(solidColorProgram, solidColorFragment);
+    gl.bindAttribLocation(solidColorProgram, 0, "pos");
+    gl.linkProgram(solidColorProgram);
+    gl.clearColor(1, 0, 0, 1);
+
+    debug('Drawing');
+    // Draw blue rectangle to backbuffer texture
+    gl.bindFramebuffer(gl.FRAMEBUFFER, backbuffer);
+    gl.useProgram(solidColorProgram);
+    gl.drawArrays(5, 0, 4);
+
+    // Blit backbuffer texture to screen
+    gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+    gl.useProgram(blendTextureProgram);
+    gl.drawArrays(5, 0, 4);
+
+    // Clear backbuffer texture to yellow
+    gl.bindFramebuffer(gl.FRAMEBUFFER, backbuffer);
+    gl.clear(gl.COLOR_BUFFER_BIT);
+
+    // Blit backbuffer texture to screen
+    gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+    gl.useProgram(blendTextureProgram);
+    gl.drawArrays(5, 0, 4);
```

Comment:
nit: should use the appropriate constant TRIANGLE_STRIP instead of 5.

Reply:
Wow, didn't realize that. Thanks, done.